rust/arrow-flight/src/arrow.flight.protocol.rs

```diff
@@ -499,9 +499,8 @@ pub mod flight_service_server {
     #[async_trait]
     pub trait FlightService: Send + Sync + 'static {
         #[doc = "Server streaming response type for the Handshake method."]
-        type HandshakeStream: Stream<
-            Item = Result<super::HandshakeResponse, tonic::Status>,
-        > + Send
+        type HandshakeStream: Stream<Item = Result<super::HandshakeResponse, tonic::Status>>
+            + Send
```

Comment:
@andygrove This seems to be some automated style that got applied when running `archery docker conda-integration`. Is it ok?

Reply:
It's a bit of an irritant because there's a slight difference in the behaviour of the formatter in the stable vs nightly versions.
We can leave it as is on this PR. I'll make time to look into a solution.